### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tgraf**
Signed-off-by: Thomas Graf <thomas@cilium.io>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# How to contribute
```
Comment (ianvernon) [inline CONTRIBUTING.md]: Very small nit: How To Contribute

**Review by ianvernon — APPROVED** [review]

**Review by aanm — APPROVED** [review]
We should also add a PR and Issue template on github

**Comment by aanm** [comment]
@tgraf I pressed the `Update branch` by mistake :(

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  * You can push individual commits to address feedback and then rebase your
+    branch at the end before merging.
+* Once the PR has been reviewed and the CI tests have passed, the PR will be
+  merged by one of the repository owners. In case this does not happen, ping us
+  on Slack.
+
+### Release Note specification
+
+If your change should be mentioned in the relesase notes, then assign one of
+the following labels to your PR or ask a project owner to do it for you:
```
Comment (aalemayhu) [inline CONTRIBUTING.md]: nit: s/relesase/release/g

**Review by aalemayhu — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (tgraf) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (tgraf) [reply CONTRIBUTING.md]: fixed

**Pull request merged**